gd32-spl-blinky-eval/platformio.ini

```diff
@@ -0,0 +1,101 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env]
+platform = https://github.com/CommunityGD32Cores/platform-gd32.git
+platform_packages =
+    framework-spl-gd32@https://github.com/CommunityGD32Cores/gd32-pio-spl-package.git
+
+[env:gd32e103v_eval]
+board = gd32e103v_eval
+framework = spl
+build_flags =  -DGD32E103V_EVAL
+
+[env:gd32e103vb_mbed]
+board = gd32e103vb_mbed
+framework = spl
+; treat as GD32E103V_EVAL to get the same 8MHz crystal setting
+; that the eval board uses. is equivalent.
+build_flags =
+    -DGD32E103V_EVAL
+
+[env:gd32103b_eval]
+board = gd32103b_eval
+framework = spl
+build_flags = -DGD32103B_EVAL
+ 
+[env:gd32103b_start]
+board = gd32103b_start
+framework = spl
+build_flags = -DGD32103B_START
+
+[env:gd32103c_eval]
+board = gd32103c_eval
+framework = spl
+build_flags = -DGD32103C_EVAL
+
+[env:gd32103c_start]
+board = gd32103c_start
+framework = spl
+build_flags = -DGD32103C_START
+
+[env:gd32103e_eval]
+board = gd32103e_eval
+framework = spl
+build_flags = -DGD32103E_EVAL
+
+[env:gd32105c_eval]
+board = gd32105c_eval
+framework = spl
+build_flags = -DGD32105C_EVAL
+
+[env:gd32105r_start]
+board = gd32105r_eval
+framework = spl
+build_flags = -DGD32105R_START
+
+[env:gd32107c_eval]
+board = gd32107c_eval
+framework = spl
+build_flags = -DGD32107C_EVAL
+
+[env:gd32303c_eval]
+board = gd32303c_eval
+framework = spl
+build_flags = -DGD32303C_EVAL
+
+[env:gd32f303ze_eval]
+board = gd32f303ze_eval
+framework = spl
+build_flags = -DGD32303Z_EVAL
+
+[env:gd32307c_eval]
+board = gd32307c_eval
+framework = spl
+build_flags = -DGD32307C_EVAL
+
+[env:gd32f307_mbed]
+board = gd32f307_mbed
+framework = spl
+
+[env:gd32350c_start]
+board = gd32350c_start
+framework = spl
+build_flags = -DGD32350C_START
+
+[env:gd32350g_start]
+board = gd32350g_start
+framework = spl
+build_flags = -DGD32350G_START_BOARD
```

Comment:
This adds `GD32350G_START_BOARD` but `gd32_includes.h` checks for `GD32350G_START`. However, since `spl.py` since very recently automatically adds board identifier macros now, it actually works.